spinalcordtoolbox/utils/validation.py

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env python
+# -*- coding: utf-8
+# Validation related utilites
+
+
+def check_dimensions_match(input_im, im_vertebral_labeling):
+    nx, ny, nz = input_im.data.shape
+    nx_vertebral, ny_vertebral, nz_vertebral = im_vertebral_labeling.data.shape
+
+    # Check dimensions consistency between the input and the vertebral labeling file
+    if (nx, ny, nz) != (nx_vertebral, ny_vertebral, nz_vertebral):
+        return False
+    else:
+        return True
```

Comment:
This could be

```suggestion
def check_dimensions_match(input_im, im_vertebral_labeling):
    return input_im.data.shape == im_vertebral_labeling.data.shape
```

Reply:
The file that you are referring to is outdated, please see the latest changes: https://github.com/neuropoly/spinalcordtoolbox/pull/3120/files#diff-745abc1c2e85edaa536f7b2eb0f8a7c18c4cc8e7db38e0e7fe5e74bee517fbd2.